test2.txt

```diff
@@ -0,0 +1 @@
+please delete
```

Comment:
I remember why this is here, but we may as well delete it. :P

Reply:
Deleted file and pushed!